index.js

```diff
@@ -30,6 +52,16 @@ class TrezorKeyring extends EventEmitter {
     this.paths = {};
     this.deserialize(opts);
     TrezorConnect.manifest(TREZOR_CONNECT_MANIFEST);
+    TrezorConnect.getFeatures().then((features) => {
```

Comment:
I think having this call right here is going to be problematic.  I'm seeing the Trezor connection window pop up way too frequently and when performing actions that don't relate to Trezor.  I'll need to research the best place to put this.

Reply:
@darkwing I addressed this in the latest commit. We can use a listener to recent Trezor device events, and the payload of those events contains the necessary data on the model